pbl

```diff
@@ -265,6 +287,12 @@ if($program eq 'pbl') {
     'help'         => sub { pbl_usage 0 },
   ) || pbl_usage 1;
 }
+else {
+  GetOptions(
+    'help'         => sub { update_bootloader_usage 0 },
+    'reinit'       => sub { unshift @todo, [ 'install' ] },
```

Comment:
I think here unshift is not needed as todo should be already empty. so not needed to unshift it as first option.

Reply:
You are right, on the other hand I don't think it is any harmless and I wanted to keep the change to the bare minimum